unity-client/Assets/Scripts/MainScripts/DCL/Controllers/BuilderMode/EntityInformation/SmartItems/Parameters/Actions/ActionEvent.cs

```diff
@@ -0,0 +1,9 @@
+using DCL.Components;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ActionEvent 
```

Comment:
I think it would be nice to have this class inside a namespace or rename it in a way that it's clear we are dealing with smart item actions